pom.xml

```diff
@@ -215,19 +216,16 @@
                         <!-- additional files like .gitignore etc.-->
                         <exclude>**/.*/**</exclude>
                         <exclude>**/*.prefs</exclude>
-                        <exclude>**/*.yml</exclude>
                         <exclude>**/*.json</exclude>
                         <exclude>**/*.log</exclude>
                         <exclude>**/*.iml</exclude>
                         <exclude>**/LICENCE</exclude>
-                        <exclude>mvnw</exclude>
-                        <exclude>mvnw.cmd</exclude>
                         <exclude>**/VERSION</exclude>
                         <exclude>**/*.md</exclude>
                         <exclude>**/dependency-reduced-pom.xml</exclude>
+                        <exclude>**/mvnw.cmd</exclude>
```

Comment:
Why you removed this?
`<exclude>mvnw</exclude>`

Basically it's the same as `mvnw.cmd` but unix version


Please return back it how was it originally
```
<exclude>mvnw</exclude>
<exclude>mvnw.cmd</exclude>
```

Reply:
Done

Reply:
@rf-jhu, sorry I still see
```
<exclude>**/mvnw</exclude>
 <exclude>**/mvnw.cmd</exclude>
```
https://github.com/riskfocus/flink-generic/blob/6ce5ef2b72810b05240250618dc1535e0f0025a8/pom.xml#L226
original version is
```
<exclude>mvnw</exclude>
<exclude>mvnw.cmd</exclude>
```

Reply:
Done